src/code/z_face_reaction.c

```diff
@@ -1,3 +1,176 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_face_reaction/func_800F1250.s")
+/**
+ * Indices of this array:
+ * - index 0x00: PLAYER_MASK_MASK_OF_TRUTH
+ * - index 0x01: PLAYER_MASK_KAFEIS_MASK
+ * - index 0x02: PLAYER_MASK_ALL_NIGHT_MASK
+ * - index 0x03: PLAYER_MASK_BUNNY_HOOD
+ * - index 0x04: PLAYER_MASK_KEATON_MASK
+ * - index 0x05: PLAYER_MASK_GAROS_MASK
+ * - index 0x06: PLAYER_MASK_ROMANIS_MASK
+ * - index 0x07: PLAYER_MASK_CIRCUS_LEADERS_MASK
+ * - index 0x08: PLAYER_MASK_POSTMANS_HAT
+ * - index 0x09: PLAYER_MASK_COUPLES_MASK
+ * - index 0x0A: PLAYER_MASK_GREAT_FAIRYS_MASK
+ * - index 0x0B: PLAYER_MASK_GIBDO_MASK
+ * - index 0x0C: PLAYER_MASK_DON_GEROS_MASK
+ * - index 0x0D: PLAYER_MASK_KAMAROS_MASK
+ * - index 0x0E: PLAYER_MASK_CAPTAINS_HAT
+ * - index 0x0F: PLAYER_MASK_STONE_MASK
+ * - index 0x10: PLAYER_MASK_BREMEN_MASK
+ * - index 0x11: PLAYER_MASK_BLAST_MASK
+ * - index 0x12: PLAYER_MASK_MASK_OF_SCENTS
+ */
+static u16 sReactionTextIds[][PLAYER_MASK_GIANTS_MASK - 1] = {
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
+    { 0 },
```

Comment:
```suggestion
static u16 sReactionTextIds[][PLAYER_MASK_GIANTS_MASK - 1] = {
    /* 0x00 */ { 0 },
    /* 0x01 */ { 0 },
    /* 0x02 */ { 0 },
    /* 0x03 */ { 0 },
    { 0 },
    /* ReactionSet 0x05 */ { 0 },
    /* ReactionSet 0x06 */ { 0 },
    /* ReactionSet 0x07 */ { 0 },
    /* ReactionSet 0x08 */ { 0 },
    { 0 },
```
I wonder if adding the reaction set number before each row would make this data more readable and navigable. I gave two different examples above of potential labelling ideas. Thoughts?

Reply:
A follow-up comment. Is there any reason this file is called `z_face_reaction.c`. I think I would like the name `z_mask_reaction.c` and `Text_GetMaskReaction` a little more just as an idea? At the risk of the overlapping terminology of `mask` as in you put on your face, and `mask` as in bit masks. But I think that's unavoidable given the importance of face masks in MM. Or even something like `z_face_mask_reaction.c` and `Text_GetFaceMaskReaction`. Just throwing out ideas, you don't have to change it if it doesn't gain traction

Reply:
I guess `z_face_reaction` was choosen because OoT has an [equivalent file](https://github.com/zeldaret/oot/blob/master/src/code/z_face_reaction.c) which does basically the same, so I based the names on the ones of that repo. We could ask them if that's an original name or not.

Reply:
For your original question: Yeah, I like the the idea of adding the reaction set number, I'll see how I can add it.

Reply:
IIRC it was inferred based on its position and the stuff on either side (`code` is alphabetical in OoT, near enough).

Reply:
Hmmm maybe it's better to keep it consistent with OoT for now and just keep it in mind for whatever we want to do when we get to full documentation mode